resources/spec.yaml

```diff
@@ -1,3 +1,18 @@
+#
```

Comment:
How is this translated to dist/spec.json?

Reply:
This will be omitted. And I just realized that with a new build we will also lose the header for `spec.yaml`. So I made additional changes.